### PR TITLE
Remove unused credentialing fields from user models

### PR DESCRIPTION
**Pull request by mscanlan-git**
This PR addresses #1571 by removing the commented code labeled "consider removing". I'm not sure if there was a consensus on whether or not to keep this code in models.py, but here's the PR!

### CODE REVIEW

**Comment by tompollard** [comment]
Thanks @mscanlan-git, let's have a chat about this. 

Making changes to the models is slightly more complicated than usual, because it requires you to include a database migration in the pull request. 

The migration is a file that explains how existing databases should be modified - in this case by deleting columns.  You may be able to generate the necessary file with:

```
python manage.py makemigrations
```

When you run `git status` you may then see the new migration file. The file would need to be added to your pull request using `git add FILENAME`, `git commit -m "Add migration file."`, then `git push`

**Comment by mscanlan-git** [comment]
> Thanks @mscanlan-git, let's have a chat about this.
> 
> Making changes to the models is slightly more complicated than usual, because it requires you to include a database migration in the pull request.
> 
> The migration is a file that explains how existing databases should be modified - in this case by deleting columns. You may be able to generate the necessary file with:

Yes, let's discuss tomorrow. I'm running into a ModuleNotFoundError and have tried a few solutions with no success.

**Comment by tompollard** [comment]
It looks like you accidentally committed one of your virtual environment files ("activate"). Please could you remove this? You can do this by undoing your last commit with:

```
git reset HEAD~
```

Then `git add` the migration file, `git commit` the change, then `git push --force` to overwrite your remote branch.

**Comment by tompollard** [comment]
Minor thing but "Fixed activate file" is a slightly misleading message because your commit history no longer contains any mention of an activate file (you removed it). A better commit message would be something like "Add database migration", which is what the commit is doing.

**Comment by bemoody** [comment]
In order to remove these fields they'll also need to be removed from the *demo fixture file* (see lines 14340 to 14361 of physionet-django/user/fixtures/demo-user.json).

I think that it is also necessary to add `MIGRATE_AFTER_INSTALL = True`, in physionet-django/user/migrations/0056_remove_credentialreview_appears_correct_and_more.py:

```
...
class Migration(migrations.Migration):
    MIGRATE_AFTER_INSTALL = True

    dependencies = [
        ('user', '0055_auto_20230330_1723'),
    ]
...
```

**Comment by mscanlan-git** [comment]
Thanks @bemoody! This solved the issue. I didn't consider the demo fixture file, I guess that was causing the repeated failures every time I went to test.

File: physionet-django/user/fixtures/demo-user.json [hunk 1]
```diff
@@ -14337,28 +14337,6 @@
...
-    "course_name_provided": null,
-    "user_understands_privacy": null,
-    "user_org_known": null,
-    "user_details_consistent": null,
-    "ref_appropriate": null,
-    "ref_searchable": null,
-    "ref_has_papers": null,
-    "ref_is_supervisor": null,
-    "ref_course_list": null,
-    "ref_skipped": null,
```
Comment (tompollard) [inline physionet-django/user/fixtures/demo-user.json:14357]: keep ref_skipped

File: physionet-django/user/fixtures/demo-user.json [hunk 2]
```diff
@@ -14337,28 +14337,6 @@
...
-    "ref_knows_applicant": null,
-    "ref_approves": null,
-    "ref_understands_privacy": null,
-    "responder_comments": ""
```
Comment (tompollard) [inline physionet-django/user/fixtures/demo-user.json:14361]: keep responder comments

File: physionet-django/user/fixtures/demo-user.json [hunk 3]
```diff
@@ -14337,28 +14337,6 @@
   "model": "user.credentialreview",
   "pk": 3,
   "fields": {
-    "application": 109,
```
Comment (tompollard) [inline physionet-django/user/fixtures/demo-user.json:14340]: keep application.

File: physionet-django/user/fixtures/demo-user.json [hunk 4]
```diff
@@ -14337,28 +14337,6 @@
...
-    "status": 30,
```
Comment (tompollard) [inline physionet-django/user/fixtures/demo-user.json:14341]: keep status

**Review by tompollard — COMMENTED** [review]
There are a few fields in the fixture that we should keep. Added inline comments to highlight them.

**Comment by tompollard** [comment]
Thanks @mscanlan-git, this looks great. Before we merge, it would be nice to clean up the commits. Right now we have ~9 commits, most of which aren't needed (they are simply changes that were made during the development process). 

As we've made a few changes to the dev branch since you started your work, we could also use this as an opportunity to rebase on the dev branch. This just means taking your new changes and putting them on top of the latest version of the `dev` branch.

If possible, please could you:

## Rebase your branch on `dev`

1. Switch to the `dev` branch and pull down the latest version:

`git switch dev`
`git pull`

2. Switch back to your `model-removals` branch:

`git switch model-removals`

3. Rebase your `model-removals` on the `dev` branch.

`git rebase dev`

## Cleaning up your commits (squashing 9 commits to 2 commits)

4. . You should be able to reset your branch to the last commit on the `dev` branch. This removes your commits, but leaves the changes to the files in your working directory.

`git reset adbde5e687a45b72de2d4fbfbb1b25b4574bf8f9`

5. Now you can do the standard `git add` and `git commit` steps to add and commit your files.

`git add physionet-django/user/fixtures/demo-user.json`
`git add physionet-django/user/models.py`
`git commit -m "remove unused fields from credentialing review model."`

`git add physionet-django/user/migrations/0056_remove_credentialreview_appears_correct_and_more.py`
`git commit -m "add migration"`

6. Now you can push the changes to the remote branch. You'll need to force push, because you'll be overwriting your previous commits.

`git push --force`

**Comment by mscanlan-git** [comment]
Thanks @tompollard! No issues squashing those commits

**Pull request merged**